.github/API/middlewares.md

```diff
@@ -8,10 +8,6 @@ This is a built-in middleware function in Opine. It parses incoming requests wit
 
 Returns middleware that only parses JSON and only looks at requests where the `Content-Type` header matches the `type` option. This parser accepts any Unicode encoding of the body and supports automatic inflation of `gzip` and `deflate` encodings.
 
```

Comment:
```suggestion

After the middleware, the existing `body` property on the `request` object (i.e. `req.body`) is overwritten with the parsed data, or an empty object (`{}`) if there was no body to parse, the `Content-Type` was not matched, or an error occurred. The original, unparsed request `body` property is still available on the `raw` property (i.e. `req.raw`).

> As `req.body`’s shape is based on user-controlled input, all properties and values in this object are untrusted and should be validated before trusting. For example, `req.body.foo.toString()` may fail in multiple ways, for example `foo` may not be there or may not be a string, and `toString` may not be a function and instead a string or other user-input.
```

It would be good to keep a reasonably explicit explanation 😄